hypothesis_bio/hypothesis_bio.py

```diff
@@ -214,21 +215,55 @@ def cds(
 
 
 @composite
-def parsed_fasta(
-    draw, comment_source: SearchStrategy = None, sequence_source: SearchStrategy = None
-) -> Dict[str, str]:
+def fasta(
+    draw,
+    comment_source: SearchStrategy = None,
+    sequence_source: SearchStrategy = None,
+    wrap_lines=True,
+    wrap_length: Optional[int] = None,
```

Comment:
You could remove `wrap_lines` and just have `wrap_length`. Then if `wrap_length` is `0` there is no line wrapping? That would also save you having to do the check for if someone has added a wrap length but said False for wrap_lines